csharp/src/Drivers/BigQuery/BigQueryConnection.cs

```diff
@@ -607,17 +608,24 @@ public override IArrowArrayStream GetInfo(List<AdbcInfoCode> codes)
             ArrowBuffer.BitmapBuilder nullBitmapBuffer = new ArrowBuffer.BitmapBuilder();
             int length = 0;
 
-            string query = string.Format("SELECT * FROM `{0}`.`{1}`.INFORMATION_SCHEMA.CONSTRAINT_COLUMN_USAGE WHERE table_name = '{2}' AND constraint_name = '{3}'",
-               Sanitize(catalog), Sanitize(dbSchema), Sanitize(table), Sanitize(constraintName));
+            // table_name = '{2}' AND
```

Comment:
These are commented-out because they might be re-enabled at some point e.g. when some other problem is fixed? If so, consider leaving an explanatory comment.